e2e/pages/adf/content_services/search/dateRangeFilterPage.ts

```diff
@@ -0,0 +1,179 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Util = require('../../../../util/util');
+import { element, by, browser, protractor } from 'protractor';
+import { DatePickerPage } from '../../material/datePickerPage';
+
+export class DateRangeFilterPage {
+
+    createdRangeFilter = element(by.css('mat-expansion-panel[data-automation-id="expansion-panel-Created Date (range)"]'));
```

Comment:
This locator is part of demo-shell, shouldn't be in this file. Please move it to searchFiltersPage.ts